src/main/io/gps.c

```diff
@@ -994,6 +1055,11 @@ typedef struct {
     ubx_nav_svinfo_channel channel[16];         // 16 satellites * 12 byte
 } ubx_nav_svinfo;
 
+typedef struct {
+    uint8_t clsId;               // Class ID of the acknowledged message 
```

Comment:
Why not `classId`, `messageId`, and an indicative name for the `struct`? This will save you having to add comments to make this understandable.

Reply:
This also comes from the documentation and it follows how the other structs are named. I'm not sure in this particular case if it's a good idea to change the naming convention.

Reply:
Ok, that's fair enough.